components/LandingPage.tsx

```diff
@@ -7,14 +7,31 @@ export default function LandingPage() {
     <VStack>
       <Box p={2} color='teal.100'>
         <Heading size='lg'>About ecoDrips</Heading>
-        <Text py={2}>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</Text>
+        <Text py={2}>Welcome to ecoDrips. We are a community of crypto enthusiasts who believe we can positively change the future of our planet through consistent and regular crypto contributions.</Text> 
+
+        <Text py={2}>Two significant challenges non-profit organizations face are:</Text>
+         
+         <Text py={2}><strong>Receiving irregular donations</strong></Text> 
+         <Text py={2}><strong>Receiving varying dollar amounts</strong></Text>
+
+         <Text py={2}>As a decentralized charity, our focus is to solve these two problems through a drip campaign where individuals can commit to contributing a small reoccurring amount of 10 DAI to their favorite non-profit organization.</Text>
+        
         <Heading size='lg'>How It Works</Heading>
-        <Text py={2}>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</Text>
-        <Text py={2}>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</Text>
+        <Text py={2}>If you have been thinking about donating crypto but are not sure where to start, this is an excellent place to begin.</Text>
+        <Text py={2}><strong>First Verify that your crypto wallet supports DAI</strong></Text>
```

Comment:
To utilize ChakraUI’s amazing capabilities, can you use `<Text as=‘strong’ ...` and remove the strong tags from these elements?